.github/workflows/quemb_unittest.yml

```diff
@@ -40,12 +40,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip
 
```

Comment:
I had to get rid of caching completely because it did not properly pick up a library that I knew was changed.

Reply:
What we can easily do and probably should do, is to reuse the installed environment from the `analysis` step when running the actual test suite. This does not require complicated haching mechanisms and cache invalidations, but we just store and reuse the installed environment when going from `analysis` to `test-suite`.

Reply:
Great :D